src/lib/components/End/index.js

```diff
@@ -11,22 +11,32 @@ const End = ({
   },
   handleChange,
 }) => (
-  <div>
-    <h2>End</h2>
-    <select
-      name="end.mode"
-      className="form-control"
-      value={mode}
-      onChange={event => handleChange(event)}
-    >
-      <option value="Never">Never</option>
-      <option value="After">After</option>
-      <option value="On date">On date</option>
-    </select>
+  <div className="px-3">
+    <div className="form-group row">
+      <div className="col-sm-2 text-sm-right">
+        <label className="col-form-label">
```

Comment:
Please connect labels with form fields, for accessibility and other features.

https://www.w3.org/WAI/tutorials/forms/labels/

Same goes for inputs without labels, please improve accessibility for them.